jedi: Disable discover_buildout_paths

This slows down jedi a lot and can make it crash on some scripts.

See also https://github.com/davidhalter/jedi/issues/1325

---
 extension/pythonFiles/lib/jedilsp/jedi/inference/sys_path.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extension/pythonFiles/lib/jedilsp/jedi/inference/sys_path.py b/extension/pythonFiles/lib/jedilsp/jedi/inference/sys_path.py
index 062a0aa..7b5fd42 100644
--- a/extension/pythonFiles/lib/jedilsp/jedi/inference/sys_path.py
+++ b/extension/pythonFiles/lib/jedilsp/jedi/inference/sys_path.py
@@ -137,6 +137,7 @@ def check_sys_path_modifications(module_context):
 
 def discover_buildout_paths(inference_state, script_path):
     buildout_script_paths = set()
+    return buildout_script_paths
 
     for buildout_script_path in _get_buildout_script_paths(script_path):
         for path in _get_paths_from_buildout_script(inference_state, buildout_script_path):
